### PR TITLE
Change data names as per para (4) of update document.

### PR DESCRIPTION
**Pull request by jamesrhester**
As per committee-agreed update document provided by @brantonc , two data names should be renamed and the previous versions deprecated. This update does that.

### CODE REVIEW

**Comment by jamesrhester** [comment]
@vaitkus - if you could check that the deprecation + aliasing has been done correctly, that'd be great. I've included @brantonc as a reviewer as well but will merge if he doesn't have time, as I'm happy that the document he provided has been implemented correctly.

File: cif_mag.dic [hunk 1]
```diff
@@ -3097,7 +3129,33 @@ save_
...
+      _description_example.case
+         1.1.1
+         32.5.122
+
+save_
+
+save_space_group_magn.point_group_number_litvin
+
+    _definition.id                '_space_group_magn.point_group_number_Litvin'
+    _alias.definition_id          '_space_group_magn.point_group_number'
```
Comment (vaitkus) [inline cif_mag.dic:3157]: We probably do not want to have `_space_group_magn.point_group_number` both as an alias and as a separate deprecated data item? Actually, if the overall purpose is to rename the item while retaining the old name as an deprecated alias we could probably avoid having almost duplicate definitions and make use of the `_alias.deprecation_date` attribute.

If I understand correctly, this attribute can be used to deprecate an alternative name without needing a separate definition for that alternative data name. See for example the `_space_group_symop_magn_ssg_operation.id` data item where the `_space_group_symop_magn_ssg.id` data name was deprecated in such a way.

File: cif_mag.dic [hunk 2]
```diff
@@ -2996,7 +2996,39 @@ save_
...
+         "4 m m"
+         "4' m' m"
+         "4' m m'"
+
+save_
+
+save_space_group_magn.point_group_name_h-m
+
+    _definition.id                '_space_group_magn.point_group_name_H-M'
+    _alias.definition_id          '_space_group_magn.point_group_name'
```
Comment (vaitkus) [inline cif_mag.dic:3030]: See https://github.com/COMCIFS/magnetic_dic/pull/60/files#r1458672040

**Comment by brantonc** [comment]
Makes sense to me.
Branton

File: cif_mag.dic (hunk 1 above)
Comment (jamesrhester) [reply cif_mag.dic:3181]: Agree. I'd forgotten about that. I will update the PR for these two data names.

**Pull request merged**